include/arch/arm/cortex_m/mpu/nxp_mpu.h

```diff
@@ -121,6 +122,77 @@
 
 #define REGION_BACKGROUND_ATTR	MPU_REGION_SU_RW
 
+
+/* Typedef for the k_mem_partition attribute*/
+typedef u32_t k_mem_partition_attr_t;
```

Comment:
This is not a structure.

Reply:
Correct. I have defined k_mem_partition_attr_t as a single-element stuct for ARMv7 and dual-element struct for ARMv8.
I did not update anything for the nxp mpu; I just re-organized the code.